core/services/relay/evm/chain_reader.go

```diff
@@ -141,6 +142,10 @@ func (cr *chainReader) Name() string { return cr.lggr.Name() }
 
 // Start registers polling filters if contracts are already bound.
 func (cr *chainReader) Start(ctx context.Context) error {
+	if cr.StateMachine.Ready() == nil {
```

Comment:
maybe upgrade to services.Engine as well

Reply:
@ilija42 Want me to do this here or in a separate ticket?

Reply:
Sure

Reply:
Created: https://smartcontract-it.atlassian.net/browse/BCI-4120